bootstrap/main.go

```diff
@@ -56,6 +56,9 @@ func mainImpl() {
 		"weave_cloud_hostname": opts.WCHostname,
 	})
 
+	// Due to some users Kubernetes clusters having unvalid certificates,
```

Comment:
*i*nvalid.

IIRC you mentioned that the usual (only observed?) problem here is self-signed certs, so perhaps mention that. "... invalid, e.g. self-signed, certificates", perhaps.